### ISSUE TITLE
SSEtoRserve specific functions are not recognized by Qlik Sense

### ISSUE THREAD

**Issue by anafotina** [qlik-oss/sse-r-plugin#8]
### Description

Hi, I've got Rserve and SSEtoRServe finally running, but it seems that Qlik Sense does not recognize the plugin specific functions such as R.ScriptAggrExStr and therefore the objects (i.e. decisiontree) are empty. Is there any way to troubleshoot / fix the problem?

### Steps to Reproduce

1. Download R and RStudio
2. Install Rserve and other needed packages
3. Download and build SSEtoRserve (with Visual Studio)
4. Configure Qlik Sense Desktop for acces to the plugin (Settings.ini)
5. Download needed Qlik extensions (i.e. decisiontree)
6. Open the sample apps (Decision Tree TItanic and other)

##### Expected behavior
I expect to see the decisin tree. 

##### Actual behavior
The objects are empty. 
The data is loaded. The function script is in the object but is not read and Qlik View does not recognize it. It says there is an error in expression. 

### Environment
Qlik Sense Desktop, June 2017 release

##### Operating system where the SSE plugin is deployed
```
[x] Windows 10
[ ] Linux
```
##### Qlik Sense
```
[x] Desktop
[ ] Server
```
### Versions

* sse-r-plugin:  v1.0.0
* Qlik Sense: Desktop, June 2017
* Operating system: Windows 10
* [Other relevant versions]: Visual Studio 2017

**Comment by josefinestal** [comment]
Hi @anafotina ! 
Thank you for input. I tried the example myself just now and it works out of the box for me, I think there is something wrong with your setup that's causing this error for you. Please check the following things:
1. First, is Rserve and SSEtoRserve.exe running before you start Sense Desktop?  
![image](https://user-images.githubusercontent.com/19163466/28610873-33cdd21e-71ea-11e7-9745-7c0fefadead3.png)
Somthing similar to the above picture should be visible in the SSEtoRserve.exe output before the engine service is started. 
2. Does the error say anything else than that there is an error in the expression? (press the information symbol in the bottom left corner ) 
![image](https://user-images.githubusercontent.com/19163466/28611014-a9e4fce8-71ea-11e7-959b-4b0b318ba009.png)
Usually you can see if e.g. the function is not valid. This information will help us solving your issue. 
3. How does your configuration look like in Settings.ini?  
4. Is the decisiontree extension added to your Sense Desktop installation? 
5. Have you tried another r-example app, for instance the R-basic example? Did they work as expected? 


Let me know about the bullet points above!

Kind regards, 
Josefine

**Comment by anafotina** [comment]
Dear @josefinestalqlik ,

Thank you very much for your reply! I managed to fix it thanks to your questions. The problem was with Settings.ini file. I changed it using CodeWriter as I had problems accessing Notepad and apparently it did not make any effect as even if the line is there in CodeWriter, it is still abscent if you open the file with Notepad. I added the line using Notepad and now it works!

Thank you,

Best regards,
Ana
